cmd/dcrinstall/dcrinstall.go

```diff
@@ -427,6 +433,34 @@ func (c *ctx) walletDBExists() bool {
 		exist(filepath.Join(dir, netSim, walletDB))
 }
 
+func (c *ctx) clientFileExists(app, filename string) bool {
+	dir := dcrutil.AppDataDir(app, false)
+	return exist(filepath.Join(dir, filename))
+}
+
+func (c *ctx) generateClientCerts(version string) error {
+	// Create certificate for politeiavoter
+	gencertsExe := filepath.Join(c.s.Destination,
+		"decred-"+c.s.Tuple+"-"+version, "gencerts")
+	piDir := dcrutil.AppDataDir("politeiavoter", false)
+	piClientCert := filepath.Join(piDir, "client.pem")
+	o, err := exec.Command(gencertsExe, piClientCert,
+		filepath.Join(piDir, "client-key.pem")).CombinedOutput()
+	if err != nil {
+		panic(string(o))
+		return err
```

Comment:
i would print this before returning the error, or wrap the output with the error

Reply:
Fixed.